tools/jarjar/jarjar.bzl

```diff
@@ -23,7 +23,7 @@ def _jarjar_library(ctx):
     jar_files = depset(transitive = [jar.files for jar in ctx.attr.jars]).to_list()
 
     command = """
-  JAVA_HOME=$(pwd)/{java_home} # this is used outside of the root
+  JAVA_HOME=$(cd "{java_home}" && pwd -P) # this is used outside of the root
```

Comment:
Do we need `-P`? Could it cause problems even?

Reply:
Yes, let me push a follow-up commit after I verify that it can work without that switch.

We've found it to be quite robust across distributions (Centos, Ubuntu, OS X) but it's definitely worth trying to minimise it if possible

Reply:
PR updated - removing `-P`.